image/types/src/lib.rs

```diff
@@ -290,7 +290,7 @@ pub struct OwnerSignedData {
     /// Owner End Date [ASN1 Time Format] For LDEV-Id certificate: Takes Preference over vendor end date
     pub owner_not_after: [u8; 15],
 
-    reserved: [u8; 2],
+    reserved: [u8; 10],
```

Comment:
Should vendor-signed data have the same? So far the approach as been "Allow the vendor to specify the field and optionally allow the owner to overwrite it"

Reply:
Done.